src/cheri_prelude_128.sail

```diff
@@ -52,6 +52,10 @@ let  cap_len_width  = sizeof(cap_len_width)
 type cap_mantissa_width : Int = 14
 let  cap_mantissa_width = sizeof(cap_mantissa_width)
 
+/* For CLoadTags and CClearTags, which operate at cacheline granularity */
+type caps_per_cacheline : Int = 8
```

Comment:
All our implementations have 512b cache lines, so 4 caps per line on CHERI-128.

Reply:
I was about to say "there's no particular reason that Sail has to match any particular implementation" but I guess we want that for TestRIG.  Maybe the answer is that there should be a WARL csr field somewhere and sail can actually use that value while hardware hard-codes it.

Reply:
It's something for riscv-config when that materialises any year now, just like all the other bits of implementation-defined behaviour.